FirebaseRemoteConfig/Sources/RCNConfigRealtime.m

```diff
@@ -384,24 +385,25 @@ - (void)fetchLatestConfig:(NSInteger)remainingAttempts targetVersion:(NSInteger)
     [strongSelf->_configFetch
         fetchConfigWithExpirationDuration:0
                         completionHandler:^(FIRRemoteConfigFetchStatus status, NSError *error) {
-                          NSLog(@"Fetching new config");
                           if (status == FIRRemoteConfigFetchStatusSuccess) {
                             if ([strongSelf->_configFetch.templateVersionNumber integerValue] >=
                                 targetVersion) {
-                              NSLog(@"Executing callback delegate");
                               for (RCNConfigUpdateCompletion listener in strongSelf->_listeners) {
                                 listener(nil);
                               }
                             } else {
-                              NSLog(
+                              FIRLogDebug(
+                                  kFIRLoggerRemoteConfig, @"I-RCN000016",
                                   @"Fetched config's template version is the same or less then the "
```

Comment:
actually here the version number is strictly < current version right? can we just say something like "Fetched config is outdated, re-fetching"